src/lib/assets/index.ts

```diff
@@ -40,8 +40,8 @@ export class Assets {
 
   zarfYaml = (path: string) => zarfYaml(this, path);
 
-  allYaml = async () => {
+  allYaml = async (rbacMode: string = "") => {
```

Comment:
I think I remember us talking about how the only two acceptable rbacMode values would be `admin` or `scoped`, right -- it's what we're enforcing in the cli cmds, right?   And, since we've decided that the default-right-now behavior would be considered `admin`, I wonder what the utility of setting a default value of empty string here would be?  As is, this implies that empty string is a valid value but is it?

I would sort have expected this to have a default value of "admin" (if we wanted to push the validation into the lib code) or to not have a default at all (if we wanted to just rely on the cli invocation).